src/create_helpers.ts

```diff
@@ -1,67 +1,77 @@
 import * as ts from 'typescript';
 import { warn } from './logger';
-import { resolveType, createFunctionParams, createFunctionReturnType, resolveTypeParameters, resolveHeritageClauses } from './type_resolve_helpers';
+import {
+    resolveType,
+    createFunctionParams,
+    createFunctionReturnType,
+    resolveTypeParameters,
+    resolveHeritageClauses,
+} from './type_resolve_helpers';
 
 const declareModifier = ts.createModifier(ts.SyntaxKind.DeclareKeyword);
 const constModifier = ts.createModifier(ts.SyntaxKind.ConstKeyword);
 const readonlyModifier = ts.createModifier(ts.SyntaxKind.ReadonlyKeyword);
 
-function validateClassLikeChildren(children: ts.Node[] | undefined, validate: (n: ts.Node) => boolean, msg: string)
-{
+function validateClassLikeChildren(
+    children: ts.Node[] | undefined,
+    validate: (n: ts.Node) => boolean,
+    msg: string,
+) {
     // Validate that the children array actually contains type elements.
     // This should never trigger, but is here for safety.
-    if (children)
-    {
-        for (let i = children.length - 1; i >= 0; --i)
-        {
+    if (children) {
+        for (let i = children.length - 1; i >= 0; --i) {
             const child = children[i];
-            if (!validate(child))
-            {
-                warn(`Encountered child that is not a ${msg}, this is likely due to invalid JSDoc.`, child);
+            if (!validate(child)) {
+                warn(
+                    `Encountered child that is not a ${msg}, this is likely due to invalid JSDoc.`,
+                    child,
+                );
```

Comment:
Maybe there is a max line length that is too short, or a minimum required params to make it do this? I don't mind splitting params to different lines, but this is happening too often.

Reply:
Depending the outcome of other comments, if this is the only this stopping us going forward, I can create a branch with a line length of 100 to compare.